src/main/java/com/microsoft/azure/vmagent/AzureVMAgentTemplate.java

```diff
@@ -384,6 +389,10 @@ public int hashCode() {
 
     private String licenseType;
 
+    @SuppressFBWarnings(value = "SE_BAD_FIELD", justification = "They are serializable in practice")
```

Comment:
Not 100% confident on this, but I checked all the node properties in core and they were all serialisable.

This will be popping up because the plugin must serialise this class when its keeping track of deployments.
I think this will be ok and in my testing I didn't see any issues with it